microservices/nris_api/openshift/_python36_oracle.dc.json

```diff
@@ -69,6 +69,14 @@
       "name": "MEMORY_REQUEST",
       "value": "768Mi"
     },
+    {
+      "name": "UWSGI_PROCESSES",
+      "value": "2"
+    },
+    {
+      "name": "UWSGI_THREADS",
+      "value": "4"
```

Comment:
Were these intentionally flipped? It was 4 processes and 2 threads before

Reply:
Yes! I did some more digging and it turns out the flow is as follows:
Since python is single threaded, the more processes you have, the more copies of your program it creates and thus requires more CPU. Basically each fork gets its own GIL and own state and everything.
For dev it seemed okay to work with one processes so i toned that down. But for Test and PROD I left em at 2

I kept threads to 2 since threads take more memory and incoming requests can still be served "multi-threaded"

Reply:
Makes sense. Thanks for the explanation 👍